plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveFileMetastore.java

```diff
@@ -85,4 +87,64 @@ public void testInsertOverwriteUnpartitioned()
     {
         // FileHiveMetastore has various incompatibilities
     }
+
+    @Override
```

Comment:
It's a bit confusing to me that we've changed FileHiveMetastore but still ended up with test overrides here.